src/wallet/wallet.h

```diff
@@ -488,6 +489,11 @@ class CWalletTx
      */
     int nIndex;
 
+    /* Refers to height of block against which tx is merkle branch linked to
```

Comment:
Nits:

1. Consider rewording for succinctness and clarity: Height of block against which tx is linked via a merkle branch.
2. End each sentence with a period as they tend to blend when reading the comment.